itests/pom.xml

```diff
@@ -266,6 +266,7 @@
                         <spring41.version>${spring41.version}</spring41.version>
                         <spring42.version>${spring42.version}</spring42.version>
                         <spring43.version>${spring43.version}</spring43.version>
+                        <spring50.version>${spring50.version}</spring50.version>
```

Comment:
The `ci-build-profile` profile has to be updated as well else it fails on Jenkins.

Reply:
This property needs to be in `ci-build-profile` as well.